contrib/libgbinder/template.py

```diff
@@ -0,0 +1,29 @@
+pkgname = "libgbinder"
+pkgver = "1.1.14"
+pkgrel = 0
+build_style = "makefile"
+make_install_target = "install-dev"
+make_check_target = "test"
+make_use_env = True
+hostmakedepends = [
+    "clang",
+    "pkgconf",
+]
+makedepends = ["libglibutil-devel", "linux-headers"]
+pkgdesc = "GLib-style interface to binder"
+maintainer = "tulilirockz <tulilirockz@outlook.com>"
+license = "BSD-3-Clause"
+url = "https://github.com/mer-hybris/libgbinder"
+source = f"{url}/archive/refs/tags/{pkgver}.tar.gz"
+sha256 = "106ab4c6605d518f6d0e969ce9f2676da3073ac7b2ed42bd89a5895833a1ae05"
+# check: race condition with include in make that makes it so it segfaults randomly when testing (trying to strace fixes the race condition)
```

Comment:
i can't reproduce this at all but can you reproduce it with make_check_args = ["-j1"]

Reply:
latest update fixes it... i literally just got the wrong version